src/components/Buttons/SecondaryButton.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+SecondaryButton.propTypes = {
+  disabled: PropTypes.bool,
+  handleClick: PropTypes.func,
+  children: PropTypes.object.isRequired,
+}
+
+export default function SecondaryButton({ handleClick, children }) {
+  return (
+    <ButtonStyled onClick={handleClick} data-testid="button">
+      {children}
+    </ButtonStyled>
+  )
+}
+
+const ButtonStyled = styled.button`
+  background-color: var(--primary-dark);
+  border: none;
+  fill: var(--white);
+  font-size: 0.8em;
+  padding: 10px;
+  height: 100%;
+  width: 36px;
+`
```

Comment:
1)
The property names of some of your components are not common practice. 
In HTML, a button has `<button onclick="" />`, in React it's  `<button onClick={} />`.
In a styled component, it's similar to React `<Button onClick={} />`.

But in your case, it's `<SecondaryButton handleClick={} />`.
It's highly recommended following these naming convention and rename it to `<SecondaryButton onClick={} />`.
This helps tools like [Storybook](https://storybook.js.org/docs/react/essentials/actions) to automatically detect event handlers because they all start with `on`. 

2)
You can directly export styled components. 
`SecondaryButton` is nearly similar to `ButtonStyled`:

```suggestion
import styled from 'styled-components/macro'

const SecondaryButton= styled.button`
  background-color: var(--primary-dark);
  border: none;
  fill: var(--white);
  font-size: 0.8em;
  padding: 10px;
  height: 100%;
  width: 36px;
`

export default SecondaryButton;
```
See another example: https://github.com/jamarob/repo-dashboard/pull/2#discussion_r532180891

3)
`data-testid` should be unique, so don't add it to a reusable component.

Reply:
Thanks for your detailed feedback! I will correct that.